### PR TITLE
Config for special attack orb plugin

### PR DESCRIPTION
**Pull request by DikkeNoob**
Includes option to change background color, recharge color and recharge opacity

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbConfig.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2017, Adam <Adam@sigterm.info>
```
Comment (sethtroll) [inline runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbConfig.java]: This should be you

File: runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java [hunk 2]
```diff
@@ -56,21 +56,23 @@
 	private static final int RUN_ORB_X_RESIZABLE = 12;
 	private static final int RUN_ORB_Y_RESIZABLE = 103;
 
-	private static final Color SPECIAL_ORB_BACKGROUND_COLOR = new Color(51, 102, 255);
-	private static final Color SPECIAL_ORB_RECHARGE_COLOR = new Color(153, 204, 255, 50);
+	private static Color SPECIAL_ORB_BACKGROUND_COLOR;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java]: These two fields should be non-static and also probably not uppercase as they are no longer final.

File: runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java [hunk 3]
```diff
@@ -56,21 +56,23 @@
 	private static final int RUN_ORB_X_RESIZABLE = 12;
 	private static final int RUN_ORB_Y_RESIZABLE = 103;
 
-	private static final Color SPECIAL_ORB_BACKGROUND_COLOR = new Color(51, 102, 255);
-	private static final Color SPECIAL_ORB_RECHARGE_COLOR = new Color(153, 204, 255, 50);
+	private Color OrbBackgroundColor;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java]: camelCase

File: runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java [hunk 4]
```diff
@@ -56,21 +56,23 @@
...
+	private Color OrbRechargeColor;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java]: camelCase

**Comment by Adam-** [comment]
This plugin mostly went away due to Jagex adding their own orb. Reopen if this gets updated.

**Pull request closed without merging**